core/amber/project/plugins.sbt

```diff
@@ -1,7 +1,7 @@
 addSbtPlugin("org.scalameta" % "sbt-scalafmt" % "2.4.2")
 addSbtPlugin("ch.epfl.scala" % "sbt-scalafix" % "0.11.0")
 
-// for scalapb code gen
-addSbtPlugin("com.thesamet" % "sbt-protoc" % "1.0.3")
 libraryDependencies += "com.thesamet.scalapb" %% "compilerplugin" % "0.11.1"
```

Comment:
is this dependency still needed? if so please put it together with the new sbt-fs2-grpc plugin.

Reply:
removed this dependency.